doc/man/taskrc.5.in

```diff
@@ -303,6 +303,7 @@ control specific occasions when output is generated. This list may contain:
     label      Column labels on tabular reports
     new-id     Provides feedback on any new task with IDs (and UUIDs for new tasks with ID 0, such as new completed tasks).
     new-uuid   Provides feedback on any new task with UUIDs. Overrides new-id. Useful for automation.
+    news-nag   Reminds to read new release highlights until the user runs "task news".
```

Comment:
Maybe we should simply call this `news`?

Reply:
Done.